inlineplz/interfaces/__init__.py

```diff
@@ -5,6 +5,16 @@
 
 from inlineplz.interfaces.github import GitHubInterface
 
+STASH_SUPPORTED = False
+try:
+    from inlineplz.interfaces.stash import StashInterface
```

Comment:
Because this internal dependency has an external dependency (stashy) that does not support python 3.5.